src/modules/Dashboard/pages/RespondentDetails/RespondentDetails.hooks.tsx

```diff
@@ -0,0 +1,48 @@
+import { generatePath, useParams } from 'react-router-dom';
+
+import { Svg } from 'shared/components/Svg';
+import { page } from 'resources';
+
+export const useRespondentDetailsTabs = () => {
```

Comment:
The secondary navigation (tabs) for Respondent Details is not in scope for [M2-5441](https://mindlogger.atlassian.net/browse/M2-5441); I believe that's what [M2-5442](https://mindlogger.atlassian.net/browse/M2-5442) is supposed to cover, if I'm not mistaken.

Since you've already implemented the tabs, you might as well assign yourself to M2-5442, and also be sure to include it in this PR's title and description.